docs/usage/index.md

```diff
@@ -0,0 +1,91 @@
+![Renovate banner](https://app.renovatebot.com/images/whitesource_renovate_660_220.jpg){ loading=lazy }
+
+# Renovate documentation
+
+Automated dependency updates.
+Multi-platform and multi-language.
+
+## Why use Renovate?
+
+<!-- markdownlint-disable list-marker-space -->
+<!-- prettier-ignore-start -->
+
+<div class="grid cards" markdown>
+
+-   :octicons-git-pull-request-24:{ .lg .middle } __Automatic updates__
+
+    ---
+
+    Get pull requests to update your dependencies and lock files.
+
+-   :octicons-calendar-24:{ .lg .middle } __On your schedule__
+
+    ---
+
+    Reduce noise by scheduling when Renovate creates PRs.
+
+-   :octicons-package-24:{ .lg .middle } __Works out of the box__
+
+    ---
+
+    Renovate finds relevant package files automatically, including in monorepos.
+
+-   :octicons-goal-24:{ .lg .middle } __How you like it__
+
+    ---
+
+    You can customize the bot's behavior with configuration files.
+
+-   :octicons-share-24:{ .lg .middle } __Share your configuration__
+
+    ---
+
+    Share your configuration with ESLint-like config presets.
+
+-   :octicons-sync-24:{ .lg .middle } __Out with the old, in with the new__
+
+    ---
+
+    Get replacement PRs to migrate from a deprecated dependency to the community suggested replacement, works with _most_ managers, see [issue 14149](https://github.com/renovatebot/renovate/issues/14149) for exceptions.
+
+-   :octicons-tools-24:{ .lg .middle } __Open source__
+
+    ---
+
+    Renovate is licensed under the [GNU Affero General Public License](https://github.com/renovatebot/renovate/blob/main/license).
+
+</div>
+
+<!-- prettier-ignore-end -->
+<!-- markdownlint-enable list-marker-space -->
+
+## Supported Platforms
+
+Renovate works on these platforms:
+
+- [GitHub (.com and Enterprise Server)](../../lib/modules/platform/github/readme.md)
+- [GitLab (.com and CE/EE)](../../lib/modules/platform/gitlab/readme.md)
+- [Bitbucket Cloud](../../lib/modules/platform/bitbucket/readme.md)
+- [Bitbucket Server](../../lib/modules/platform/bitbucket-server/readme.md)
+- [Azure DevOps](../../lib/modules/platform/azure/readme.md)
+- [AWS CodeCommit](../../lib/modules/platform/codecommit/readme.md)
+- [Gitea and Forgejo](../../lib/modules/platform/gitea/readme.md)
+- [Gerrit (experimental)](../../lib/modules/platform/gerrit/readme.md)
```

Comment:
The new links work for me in the VSCode editor on Codespaces. But please double check my work! :smile:

Reply:
they are wrong 🙃

Reply:
And of course my first attempt at the links is wrong. :upside_down_face:

I changed the links, check if those are correct. :smile: